File: PRAK501-2510817220023-NURFITRIA RAMADHANI.py
def MaxBilangan(a,b,c,d):
    max=a
    if b>max:
        max=b
    if c>max:
        max=c
    if d>max:
        max=d
    return max
    
a,b,c,d=map(int,input().split())  
hasil=MaxBilangan(a,b,c,d)
print(hasil)             

File: PRAK501-2510817220023-NURFITRIA RAMADHANI.c
#include <stdio.h>

int MaxBilangan(int a,int b,int c,int d){
    int max=a;
    if(b>max){
        max=b;
    }
    if(c>max){
        max=c;
    }
    if(d>max){
        max=d;
    }
    return max;
}

int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d); 
    int hasil = MaxBilangan(a, b, c, d); 
    printf("%d", hasil); 
    return 0;
}